src/main/haskell/kore/src/Kore/Builtin/Krypto.hs

```diff
@@ -32,26 +44,24 @@ import           Data.String
 import           Data.Text
                  ( Text )
 import qualified Data.Text.Encoding as Text
-
-import           Kore.AST.MetaOrObject
-                 ( Object )
+import           Data.Word
+                 ( Word8 )
 import qualified Kore.Builtin.Builtin as Builtin
+import qualified Kore.Builtin.Int as Int
 import qualified Kore.Builtin.String as String
-import           Kore.IndexedModule.MetadataTools
-                 ( MetadataTools )
-import           Kore.Sort
-                 ( Sort )
-import           Kore.Step.Function.Data
-                 ( AttemptedFunction )
-import           Kore.Step.Pattern
-import           Kore.Step.Simplification.Data
-                 ( Simplifier, StepPatternSimplifier )
-import           Kore.Step.StepperAttributes
-                 ( StepperAttributes )
+
+import Control.Exception.Base
+       ( assert )
 
 keccakKey :: IsString s => s
 keccakKey = "KRYPTO.keccak256"
 
+ecsdaRecover :: String
```

Comment:
May I suggest,
```suggestion
ecdsaRecover :: IsString s => s
```
You will need to `import Data.String (IsString)` above, and you may delete the other definition.